webpack.config.js

```diff
@@ -23,10 +24,8 @@ module.exports = merge(baseConfig, extractCssConfig, {
       template: './www/front_src/public/index.html',
       filename: '../index.html',
     }),
+    new BundleAnalyzerPlugin(),
```

Comment:
Do not forget to remove this :)

Reply:
Actually I'm going to set it up in a "prod" configuration